wannierberri/system/sym_wann.py

```diff
@@ -93,14 +93,20 @@ def __init__(
             'AA': 1,
             'BB': 1,
             'CC': -1,
-            'SS': -1
+            'SS': -1,
+            'SH': -1,
+            'SA': 1,
+            'SHA': 1
```

Comment:
Are you sure about these values for SA, SHA? 
The comment line (by @Liu-Xiaoxiong ) says `the I-odd vectors have "+1" here`, but that's only about vectors, not scalars (Ham) or rank-2 tensors.

I would formulate it as, the sgn corresponds to the number of pseudovectors in the tensor. I mean, SS is spin, so it is pseudovector, CC has vector product, so i is pseudo vetor. SA and SHA are "direct product of spin and position", so it sghould be "-1" . If we had a direct product of two spins, or vector product of spin and A, that would be "+1".  Do you agree?
Note, in #316  put "-1" for those matrices, and it seems to work.